lib/matplotlib/container.py

```diff
@@ -18,7 +18,7 @@ def __repr__(self):
     def __new__(cls, *kl, **kwargs):
         return tuple.__new__(cls, kl[0])
 
-    def __init__(self, kl, label=None):
+    def __init__(self, kl, label=None, lorder=''):
```

Comment:
Empty string is used as it is always marked as larger then an int. If None was used, we would have the opposite effect (or an unwanted effect). Perhaps instead it should be set to infinity? What do you think?